pkg/cmd/repo/list/list.go

```diff
@@ -92,7 +92,7 @@ func NewCmdList(f *cmdutil.Factory, runF func(*ListOptions) error) *cobra.Comman
 	cmd.Flags().BoolVar(&opts.Source, "source", false, "Show only non-forks")
 	cmd.Flags().BoolVar(&opts.Fork, "fork", false, "Show only forks")
 	cmd.Flags().StringVarP(&opts.Language, "language", "l", "", "Filter by primary coding language")
-	cmd.Flags().StringVar(&opts.Topic, "topic", "", "Filter by topic")
+	cmd.Flags().StringArrayVarP(&opts.Topic, "topic", "T", nil, "Filter by topic")
```

Comment:
Note that this tries to sneak in a shorthand `-T` for the `--topic` flag, but we'd prefer that only primary (i.e. most used) flags have shorthands, and all others do not.

Also I've changed this to be StringSliceVarP because that will split values by comma:
```
gh repo list --topic cli,terminal
```

Reply:
Sorry, I forget to remove the shorthand. I added it for testing purpose ! 

For the StringSliceVarP, I saw that both were used.Didn't know if there was a preference. StringArrayVar are to be migrated?

Reply:
@kratisto It's not immediately clear, but in the underlying `pflag` library that the Cobra framework is using:

- StringArray is for flags that can be specified multiple times, e.g. `--topic foo --topic bar --topic baz`
- StringSlice is for flags that can be specified multiple times _and_ splits single values on a comma, e.g. `--topic foo,bar --topic baz`

We use StringSlice for most flags but we consciously use StringArray when we want to allow literal commas in the value itself.